mm2src/coins/z_coin.rs

```diff
@@ -1158,12 +1306,16 @@ impl MarketCoinOps for ZCoin {
     fn my_balance(&self) -> BalanceFut<CoinBalance> {
         let coin = self.clone();
         let fut = async move {
-            let sat = coin
+            let balance_sat = coin
                 .my_balance_sat()
                 .await
                 .mm_err(|e| BalanceError::WalletStorageError(e.to_string()))?;
-            Ok(CoinBalance::new(big_decimal_from_sat_unsigned(sat, coin.decimals())))
+
+            let spendable = big_decimal_from_sat_unsigned(balance_sat, coin.decimals());
+
+            Ok(CoinBalance::new(spendable))
```

Comment:
Do we not want to add the change notes from locked_notes_db as unspendable balance here?
User would see unspendable balance and learn he needs to wait for it before trying to create next transaction.